### PR TITLE
instructions for the admins to test the raids

### PR DESCRIPTION
**Pull request by MSilva95**

### CODE REVIEW

File: subjects/sudoku/audit/README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 #### Sudoku
 
+***NOTE: If you are an admin and you want to test this project, follow the instructions [HERE](https://github.com/01-edu/go-tests)***
```
Comment (kigiri) [inline subjects/sudoku/audit/README.md:3]: Did you check how this looked in the platform ?

File: subjects/quad/README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# quadrangle
+## quadrangle
```
Comment (kigiri) [inline subjects/quad/README.md:1]: why did you change this ? Is it for display reason on the platform ?

Structurally it made more sense for it to be H1, maybe we should lower the following titles to H3 instead of H2

Comment (MSilva95) [reply subjects/quad/README.md:1]: I changed it to be the same as the other projects, for consistency, but if the structure is not relevant in this case we can let it be an H1 just to distinguish the main tittle better

Comment (MSilva95) [reply subjects/quad/README.md:1]: quad-readme

This is what it will look like

**Review by kigiri — APPROVED** [review]
